aptos-move/move-examples/defi/aptoswap/core/sources/liquidity_pool.move

```diff
@@ -0,0 +1,417 @@
+module pool_manager::liquidity_pool {
```

Comment:
Can you add more documentation for this module? We should also write a tutorial to walk people through this